main.py

```diff
@@ -1021,8 +1021,6 @@ def get_redirect_route(
     get_redirect_route(
         r'%s' % feconf.BLOG_DASHBOARD_DATA_URL,
         blog_dashboard.BlogDashboardDataHandler),
-    get_redirect_route(
-        r'%s' % feconf.BLOG_DASHBOARD_URL, blog_dashboard.BlogDashboardPage),
```

Comment:
Can we remove `BLOG_DASHBOARD_URL` from feconf?

Reply:
Yes, removed It.